docs/wxm-devices/D1-bundle/D1-connect-and-claim.mdx

```diff
@@ -0,0 +1,233 @@
+  </TabItem>
+  </Tabs>
+
+  <span>
+
+The WG1200 Gateway's frequency must be set to match the frequency of your weather station so it can listen for the weather data it transmits. There are 2 frequencies available:
+- Model A: 868 MHz (default)
+- Model B: 915 MHz
+
+:::note
+**If your weather station is 868 MHz, you don't need to change anything as it is configured to this value by default.**
+If unsure, check the sticker under the station or the box that came in it.
+:::
+  </span>
+</div>
+
+### Steps to change the frequency
+<div>
+
+  <Tabs groupId="WG1200 Gateway">
+  <TabItem value="WG1200 Gateway" >
+    <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+      <img class="control" src={useBaseUrl("img/wxm-devices/D1-bundle/Image_1_Swipe-right.png")} style={{ width: '24%', height: 'auto' }} />
+      <img class="control" src={useBaseUrl("img/wxm-devices/D1-bundle/Image_2_select_freq.png")} style={{ width: '24%', height: 'auto' }} />
+      <img class="control" src={useBaseUrl("img/wxm-devices/D1-bundle/Image_3_set_freq.png")} style={{ width: '24%', height: 'auto' }} />
+      <img class="control" src={useBaseUrl("img/wxm-devices/D1-bundle/Set_Frequency.png")} style={{ width: '24%', height: 'auto'  }} />
+    </div>
+    <ol>
+        <li>Swipe right on the touch screen</li>
+        <li>Click the Frequency options and select the new frequency your WG1200 Gateway will listen to</li>
+        <li>A confirmation dialog will open. Read it carefully and click "Continue" or "Cancel" accordingly</li>
+        <li>Finally, your WG1200 Gateway will restart setting the frequency to the selected one </li>
+    </ol>
+  </TabItem>
+</Tabs>
+
+
+</div>
+
+:::caution
+If you don't use the correct frequency as your weather station, WG1000 Gateway will not be able to communicate with your weather station, thus you will not send any weather data and receive no WXM rewards!
+:::
+
+<hr/>
+
+### Connect WG1200 Gateway to WiFi 
+
+  <Tabs groupId="WG1200 Gateway">
+  <TabItem value="WG1200 Gateway WiFi Configuration" >
+    <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+      <img class="control" src={useBaseUrl("img/wxm-devices/D1-bundle/D1-Wifi.png")} style={{ width: '24%', height: 'auto', marginBottom: '20px' }} />
+      <img class="control" src={useBaseUrl("img/wxm-devices/D1-bundle/D1_waiting_connection.png")} style={{ width: '24%', height: 'auto', marginBottom: '20px' }} />
+      <img class="control" src={useBaseUrl("img/wxm-devices/D1-bundle/D1_connected.png")} style={{ width: '24%', height: 'auto', marginBottom: '20px' }} />
+       </div>
+       
+    <ol>
+        <li>Swipe right on the touch screen and select the Configure WiFi option</li>
+        <li>The WG1200 Gateway will restart to enter configuration mode.</li>
+        <li>An Access Point will be created with SSID "WXM_WG1200", you should connect to it from your smartphone.</li>
```

Comment:
Please add an explanation regarding what @liliumjsn mentioned about "it is normal to not have internet connection while connected to it."